### PR TITLE
Epic/multiple properties ga master

### PR DESCRIPTION
**Pull request by Leooo**
- [x] allow multiple ga properties to coexist (example use case: different properties for parent app vs engines)

See https://github.com/adopted-ember-addons/ember-metrics/issues/217#issuecomment-652462290

### CODE REVIEW

**Comment by Leooo** [comment]
@jherdman @poteto @chrismllr thanks

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-metrics",
-  "version": "0.16.0",
```
Comment (jherdman) [inline package.json:3]: Revert

Comment (Leooo) [reply package.json:3]: why, don't want to create a new release for that?

Comment (jherdman) [reply package.json:3]: This is up to the maintainers. Please remove.

Comment (Leooo) [reply package.json:3]: removing package-lock yes (and adding to gitignore next then) - although it is recommended to checkout package-lock in general

Comment (Leooo) [reply package.json:3]: removed package-lock and added to gitignore

Comment (Leooo) [reply package.json]: reverted package bump then

**Review by jherdman — CHANGES_REQUESTED** [review]
LGTM. One small request: would you be able to squash this into a single commit with a message describing your change?

I'll aim to release this today along with a few other small changes.

Thank you kindly for your work! ❤️

**Comment by Leooo** [comment]
lovely, I screwed up everything. Bloody force-push. Let me recreate a PR @jherdman

**Comment by Leooo** [comment]
moved to https://github.com/adopted-ember-addons/ember-metrics/pull/268 thanks

**Comment by Leooo** [comment]
`epic/last_branch_final_top_end_finished_infinity` in https://github.com/adopted-ember-addons/ember-metrics/pull/269

**Pull request closed without merging**